src/com/github/alex_moon/porcupi/views/View.java

```diff
@@ -12,13 +12,14 @@
 import com.github.alex_moon.porcupi.handlers.PokeHandler;
 import com.github.alex_moon.porcupi.handlers.Pokeable;
 import com.github.alex_moon.porcupi.manager.ManagerServer;
+import com.github.alex_moon.porcupi.messages.PokeMessage;
 import com.github.alex_moon.porcupi.responses.Response;
 import com.google.gson.Gson;
 
 public class View implements Pokeable {
     protected static Gson gson = new Gson();
+    protected Map<String, Object> track = new HashMap<String, Object>();
```

Comment:
this needs rethinking - also, tbh I reckon we SHOULD just do the reflection bit eh - this should be as transparent as possible